docker/docker-compose.yml

```
@@ -229,7 +229,7 @@ services:
       # to do that, set DB_DATA_VOLUME in your env or use a docker override file
       # For example:
       #   DB_DATA_VOLUME=./dev/db_data
-      - ${DB_DATA_VOLUME:-dbdata_pg16}:/var/lib/postgresql/data
+      - ${DB_DATA_VOLUME:-dbdata_pg17}:/var/lib/postgresql/data
       - ./tmp/dumps:/tmp/dumps
       - ./db:/mnt
     ports:
```

We should probably have a default local port. Most people probably have an override to get a static port which works as well, but is an extra step. Something like  `- "${DBPORT:-15432}:5432"`

---

@yakloinsteak do you mean like? (and drop the next line)
```suggestion
    ports:
      - "${DBPORT:-15432}:5432"
```

---

Yes.